config/router.rb

```diff
@@ -0,0 +1,27 @@
+module Router
+  ROUTES_PATH = 'config/routes/**/*.rb'
+
+  def specify(route_name)
+    extend(route_module(route_name))
+
+    call
+  end
+
+  class << self
+    def included(_)
+      routes.each(&method(:require))
+    end
+
+    private
+
+    def routes
+      Dir[Rails.root.join(ROUTES_PATH)]
+    end
+  end
+
+  private
+
+  def route_module(route_name)
+    "Routes::#{route_name.to_s.camelize}".constantize
```

Comment:
**We need to:**

1. think how to receive the scope which contains specify that has been called;
2. build class name with respective namespacing